doc/dev/config.rst

```diff
@@ -57,7 +57,7 @@ Reading configuration values
 There are two ways for Ceph code to get configuration values. One way is to
 read it directly from a variable named "g_conf," or equivalently,
 "g_ceph_ctx->_conf." The other is to register an observer that will be called
-every time the relevant configuration values changes.  This observer will be
+every time the relevant configuration values changes. This observer will be
```

Comment:
and i'd suggest avoid these sort of "minor changes" if possible unless they are piggy-backed by more significant changes. IMO, they are not typo at all. and there are not visible change introduced by this commit. please try to build the script and check the output.